<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalDataSourceWidget</class>
 <widget class="QWidget" name="LocalDataSourceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Local Data Source</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="openButton">
     <property name="text">
      <string>Open trial ...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="trialLabel">
     <property name="text">
      <string>Trial:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="trialNameLabel">
     <property name="text">
      <string>No trial opened</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="outputsLabel">
     <property name="text">
      <string>Outputs:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTableWidget" name="tableWidget">
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Selected</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Output</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>LocalDataSourceWidget</receiver>
   <slot>onOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LocalDataSourceWidget</receiver>
   <slot>onEdit(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpen()</slot>
  <slot>onEdit(QString)</slot>
  <slot>onSelectionChange(bool)</slot>
 </slots>
</ui>
